jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridViewImpl.java

```diff
@@ -115,6 +120,10 @@
     @Inject
     private DataSetEditorManager dataSetEditorManager;
 
+    public void setDataSetEditorManager(DataSetEditorManager dataSetEditorManager) {
```

Comment:
This method is not needed if it is only for testing, @InjectMocks already handles that.